### DISCUSSION TITLE
Any chance of a menu for Model 2 Emulator too?

### DISCUSSION THREAD

**Discussion by slikvik55** [BackPonBeauty/Sega-Model-3-UI-for-20240128-#40] [General]
The original UI written by the same person who wrote Super Model UI is missing from the Internet! =]

**Comment by BackPonBeauty** [comment]
from ChatGPT

Model 2 Emulator is equipped with a basic **GUI (Graphical User Interface)**. This allows users to easily adjust settings, select games, configure controllers, change video options, manage ROM paths, and more without using command lines.

This GUI is designed to make emulator operation intuitive, and even users who are not familiar with arcade games can smoothly load and configure games. However, Model 2 Emulator's GUI is limited to basic functions, and users may need to manually configure the settings separately if they want to make detailed settings or use advanced functions.

This simplicity is also the design intention to maximize the performance of the emulator and maintain light operation.



If you search for 'M2emulator_1.1a.7z' you will find many results.
Maybe older versions didn't have a GUI, but ver1.1a does, so I don't feel the need to create a new UI.

PonMi(I'm behind you)
